modules/social_features/social_like/social_like.install

```diff
@@ -18,6 +18,28 @@ function social_like_install() {
   _social_like_set_defaults();
 }
 
+/**
+ * Implements hook_entity_extra_field_info_alter().
```

Comment:
This should be in the `.module` file instead of the `.install` file.

Reply:
Damn busted, good catch 😂